postgres/README.md

```diff
@@ -1,19 +1,119 @@
-# PostgreSQL Data Accelerator
+# Postgres Data Accelerator
 
-Follow these steps to get started with PostgreSQL as a Data Accelerator.
+This quickstart will use a demo instance of Postgres. Follow the quickstart to create Postgres instance and get started with Postgres as a Data Accelerator.
```

Comment:
We should also add a sentence or two on why this is useful.

Reply:
Added one sentence explaining why accelerator is useful